vochain/start.go

```diff
@@ -644,3 +652,58 @@ func AminoKeys(key crypto25519.PrivKey) (private, public []byte, err error) {
 func AminoPubKey(pubkey []byte) ([]byte, error) {
 	return tmjson.Marshal(pubkey)
 }
+
+// ImportSnapshot imports a state snapshot given a URL and decompress it
+// if ForceResync==true overrides the node data, else download and ignore
+func ImportSnapshot(vochaincfg *config.VochainCfg) error {
```

Comment:
I think its cleaner and more elegant to not make this function dependent on vochain config. Instead is better to write atomic/standalone functions, makes the code more composable.

So ImportSnapshot() should take just the required argunemts (dataDir, URL). The caller should make the proper checks (ForceResync) and decide if the function must be executed or not.

Reply:
Just refactored the code, now I think is more modular and atomic